templates/kops/kops-private-topology.yaml.gotmpl

```diff
@@ -55,14 +55,12 @@ spec:
   {{- if bool (getenv "KOPS_AUTHORIZATION_RBAC_ENABLED" "false") }}
     anonymousAuth: false
     authorizationMode: {{ getenv "KOPS_KUBE_API_SERVER_AUTHORIZATION_MODE" "RBAC" }}
-    authorizationRbacSuperUser: {{ getenv "KOPS_KUBE_API_SERVER_AUTHORIZATION_RBAC_SUPER_USER" "admin" }}
```

Comment:
Why are we dropping this?

Reply:
Deprecated in kubernetes 1.11, not supported in 1.12. https://github.com/kubernetes/kubernetes/pull/61213